threat-intelligence-platform/info.json

```diff
@@ -0,0 +1,247 @@
+    "publisher": "spryIQ.co",
+    "icon_small_name": "threat-intelligence-platform_32x32.png",
+    "icon_large_name": "threat-intelligence-platform_80x80.png",
+    "cs_compatible": true,
+    "cs_approved": false,
+    "category": "Threat Intelligence",
+    "help_online": "https://github.com/jitesh-spryiq/connector-threat-intelligence-platform/blob/develop/docs/ThreatIntelligencePlatformConnectorDoc.md",
+    "configuration": {
+        "fields": [
+            {
+                "title": "Server URL",
+                "type": "text",
+                "description": "URL of the Threat Intelligence Platform connector to access the connector website.",
+                "tooltip": "URL of the Threat Intelligence Platform connector to access the connector website.",
+                "name": "server",
+                "required": true,
+                "visible": true,
+                "editable": true,
+                "value": "api.threatintelligenceplatform.com",
+                "placeholder": "e.g: https://api.threatintelligenceplatform.com or api.threatintelligenceplatform.com"
+            },
+            {
+                "title": "API key",
+                "type": "password",
+                "description": "Provide API token, used for user authentication.",
+                "tooltip": "Provide API token, used for user authentication.",
+                "name": "api_key",
+                "required": true,
+                "visible": true,
+                "editable": true,
+                "value": "",
+                "placeholder": "e.g:at_CjnYgbaner3wqUxhZ6aej4hAxbMqEawABEA"
+            }
+        ]
+    },
+    "operations": [
+        {
+            "title": "Get Domain Infrastructure Analysis Details",
+            "operation": "get_domain_infrastructure_analysis_details",
+            "description": "Retrieved information for a given domain name, get a collection of its web, mail, and name servers as well as its known subdomains. For each infrastructure entry, find out its IP address, geolocation, and subnetwork information.",
+            "annotation": "domain_infrastructure_analysis_details",
+            "category": "investigation",
+            "output_schema": [
+                {
+                    "domainName": "",
+                    "resourceType": "",
+                    "ipv4": "",
+                    "geolocation": {},
+                    "subnetwork": {}
+                }
+            ],
+            "parameters": [
+                {
+                    "title": "Domain Name",
+                    "type": "text",
+                    "name": "domain_name",
+                    "required": true,
+                    "visible": true,
+                    "editable": true,
+                    "value": "threatintelligenceplatform.com",
```

Comment:
you can remove the value. This is considered to be default one. Placeholder is sufficient .